Library/Homebrew/services/commands/info.rb

```diff
@@ -53,6 +53,7 @@ def self.output(hash, verbose:)
           return out unless verbose
 
           out += "File: #{hash[:file]} #{pretty_bool(hash[:file].present?)}\n"
+          out += "Registered at login: #{pretty_bool(hash[:registered])}\n"
```

Comment:
Could/should this be in `list` instead/as well?

Reply:
Probably doesn't really need to be.

The information is technically already there in `list` - if it's in `LaunchAgents` then it's registered. It's just not very good to consume as a JSON API.

Technically doesn't need to be in `info` non-JSON either but it's in the verbose section which is probably a right fit.

Reply:
My thinking of the benefits for `list` are that you can get this information for all formulae in a single call.